package.json

```diff
@@ -142,67 +143,46 @@
     "eslint-plugin-prettier": "^3.1.3",
     "globby": "^11.0.2",
     "husky": "^4.2.5",
-    "mocha": "^7.2.0",
+    "mocha": "^8.3.0",
     "npm": "^7.5.2",
     "nyc": "^15.1.0",
-    "oclif": "oclif/oclif#publish-s3-keys-compiled",
+    "oclif": "^1.16.2",
     "prettier": "^2.0.5",
     "pretty-quick": "^2.0.1",
+    "shelljs": "^0.8.4",
     "sinon": "^9.0.2",
     "ts-node": "^8.10.2",
     "typescript": "^4.1.3"
   },
   "scripts": {
-    "build": "yarn build-standalone",
-    "build-mac": "yarn build-mac-from-archive",
-    "build-mac-from-archive": "scripts/build/installers/darwin_from_archive",
-    "build-mac-from-sources": "scripts/build/installers/darwin_from_sources",
-    "build-standalone": "scripts/build/standalone",
-    "build-windows": "scripts/build/installers/windows",
-    "pack:tarballs": "oclif pack:tarballs",
-    "pack:macos": "oclif pack:macos",
-    "pack:deb": "oclif pack:deb",
-    "pack:win": "oclif pack:win",
-    "upload:tarballs": "oclif upload:tarballs",
-    "upload:macos": "oclif upload:macos",
-    "upload:deb": "oclif upload:deb",
-    "upload:win": "oclif upload:win",
-    "promote": "oclif promote",
-    "clean": "yarn clean-dist && yarn clean-test",
-    "clean-all": "scripts/clean/all",
-    "clean-build": "scripts/clean/build",
-    "clean-compile": "yarn clean-all && yarn install --pure-lockfile",
-    "clean-dist": "scripts/clean/dist",
-    "clean-test": "scripts/clean/test",
-    "compile": "scripts/build/compile",
-    "coverage-report": "scripts/test/coverage",
+    "XXpostpack:tarballs": "./scripts/verify-tarballs",
```

Comment:
Why is there the `XX`?

Reply:
It's not working =( So I don't want it to run right now.

Reply:
That was the note in my slack message, I can add a note here too.

Reply:
If only you could have commentes in package.json.